backend/server/server.go

```diff
@@ -0,0 +1,21 @@
+package server
+
+import (
+	"2022_2_GoTo_team/server/api"
+	"github.com/gorilla/mux"
+	"log"
+	"net/http"
```

Comment:
Bets-practise в гошке - сначала импорты из стандартной либы, затем внешние пакеты, затем пакеты из текущей репы

Как-то так будет

```suggestion
	"log"
	"net/http"
	
	"github.com/gorilla/mux"

	"2022_2_GoTo_team/server/api"
```